www/source/blog/2017-10-09-node-scaffolding-habitat.html.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Node Scaffolding - What's Available Today and What's Coming Soon!
+date: 2017-10-09
+author: Nell Shamrell-Harrington
+tags: scaffolding
+category: community
+classes: body-article
+---
+
+Greetings fellow Habicats!
+
+One of the most useful feature of Habitat is scaffoldings - these allow you to package your application with Habitat with very minimal code.  They enable you to very rapidly (within minutes) package an application and export it to whatever format you need.  One of the most popular Habitat scaffoldings is the NodeJS scaffolding.
```

Comment:
I'd recommend highlighting that Scaffolding makes things _even easier_ than packaging an app without it. It's also a good time to call out that Scaffolding is made available to encapsulate in a 1:1 way how to package an app for a given language runtime (if applicable) and build tool